app/services/preserved_object_handler.rb

```diff
@@ -9,12 +9,15 @@ class PreservedObjectHandler
   include ::MoabValidationHandler
   include ActiveModel::Validations
 
-  # Note: supplying validations here to allow validation before use, e.g. incoming_version in numeric logic
   validates :druid, presence: true, format: { with: DruidTools::Druid.pattern }
   validates :incoming_version, presence: true, numericality: { only_integer: true, greater_than: 0 }
   validates :incoming_size, numericality: { only_integer: true, greater_than: 0 }
   validates_each :endpoint do |record, attr, value|
-    record.errors.add(attr, 'must be an actual Endpoint') unless value.is_a?(Endpoint)
+    if value.is_a?(Endpoint)
```

Comment:
ditto on syntax:  I'm not in with this, but it gets the job done. Improvements welcome.